chart/templates/configmap.yaml

```diff
@@ -12,6 +12,11 @@ data:
     checks:
       {{- if .Values.runtimeConfig.health}}
       health:
+        interval: {{ .Values.runtimeConfig.health.interval | default 1 }}
+        timeout: {{ .Values.runtimeConfig.health.interval | default 3 }}
+        retry:
+          count: {{ .Values.runtimeConfig.health.retry.count | default 3 }}
+          delay: {{ .Values.runtimeConfig.health.retry.delay | default 1 }}
```

Comment:
Did you forget the unit here? (1s) There are also 2 other fields with possibly the same problem.

Reply:
Ah yes thanks, totally forgot to change that 😄